tests/integration/test_api.py

```diff
@@ -444,7 +449,18 @@ async def test_config_with_orgs(
                 ("test-cluster", "org1", Balance(), Quota()),
                 ("test-cluster", "org2", Balance(), Quota()),
             ],
+            cluster_user_role=ClusterUserRoleType.MANAGER,
+        )
+
+        admin_client = await admin_client_factory(regular_user.token)
```

Comment:
Could we also try to create an org without a cluster and verify whether it's included in the `/config` response?

Reply:
done.